addons/mail/static/src/models/composer/composer.js

```diff
@@ -82,6 +82,15 @@ function factory(dependencies) {
             });
         }
 
+        _getRecipients() {
+            const mentionned = this.mentionedPartners.map(partner => partner.id);
+            let suggestedPartners = [];
+            if (this.thread && this.thread.suggestedPartners) {
+                suggestedPartners = this.thread.suggestedPartners.filter(partner => partner.checked && partner.partner).map(partner => partner.partner.id);
+            }
+            return [...new Set([...mentionned, ...suggestedPartners])];;
```

Comment:
Should be list of commands, here likely
```js
return [['replace', recipients]];
```

where `recipients` is `[...new Set([...mentionned, ...suggestedPartners])]`